buildout.cfg

```diff
@@ -3,15 +3,25 @@ extends = travis.cfg
 parts =
     instance
     celery
+    test
```

Comment:
hm, maybe you should undo/revert the change to buildout.cfg (ie. you can make your buildout.cfg changes locally but don't commit them and don't push them)